pkg/thing/interactors/update_schema.go

```diff
@@ -19,33 +19,33 @@ type interval struct {
 
 // rules reference table: https://knot-devel.cesar.org.br/doc/thing/unit-type-value.html
 var rules = map[int]schemaType{
-	0x0000: schemaType{valueType: 4, unit: 0},              // RAW   => NONE
-	0x0001: schemaType{valueType: 1, unit: interval{1, 3}}, // INT   => VOLTAGE
-	0x0002: schemaType{valueType: 1, unit: interval{1, 2}}, // INT   => CURRENT
-	0x0003: schemaType{valueType: 1, unit: 1},              // INT   => RESISTENCE
-	0x0004: schemaType{valueType: 1, unit: interval{1, 3}}, // INT   => POWER
-	0x0005: schemaType{valueType: 1, unit: interval{1, 3}}, // INT   => TEMPERATURE
-	0x0006: schemaType{valueType: 1, unit: 1},              // INT   => RELATIVE_HUMIDITY
-	0x0007: schemaType{valueType: 1, unit: interval{1, 3}}, // INT   => LUMINOSITY
-	0x0008: schemaType{valueType: 1, unit: interval{1, 3}}, // INT   => TIME
-	0x0009: schemaType{valueType: 1, unit: interval{1, 4}}, // INT   => MASS
-	0x000A: schemaType{valueType: 1, unit: interval{1, 3}}, // INT   => PRESSURE
-	0x000B: schemaType{valueType: 1, unit: interval{1, 4}}, // INT   => DISTANCE
-	0x000C: schemaType{valueType: 2, unit: interval{1, 2}}, // FLOAT => ANGLE
-	0x000D: schemaType{valueType: 2, unit: interval{1, 4}}, // FLOAT => VOLUME
-	0x000E: schemaType{valueType: 2, unit: interval{1, 3}}, // FLOAT => AREA
-	0x000F: schemaType{valueType: 2, unit: 1},              // FLOAT => RAIN
-	0x0010: schemaType{valueType: 2, unit: 1},              // FLOAT => DENSITY
-	0x0011: schemaType{valueType: 2, unit: 1},              // FLOAT => LATITUDE
-	0x0012: schemaType{valueType: 2, unit: 1},              // FLOAT => LONGITUDE
-	0x0013: schemaType{valueType: 1, unit: interval{1, 4}}, // INT   => SPEED
-	0x0014: schemaType{valueType: 2, unit: interval{1, 6}}, // FLOAT => VOLUMEFLOW
-	0x0015: schemaType{valueType: 1, unit: interval{1, 6}}, // INT   => ENERGY
-	0xFFF0: schemaType{valueType: 3, unit: 0},              // BOOL  => PRESENCE
-	0xFFF1: schemaType{valueType: 3, unit: 0},              // BOOL  => SWITCH
-	0xFFF2: schemaType{valueType: 4, unit: 0},              // RAW   => COMMAND
-	0xFF10: schemaType{valueType: 1, unit: 0},              // INT   => ANALOG
-	0xFFFF: schemaType{valueType: 4, unit: 0},              // RAW   => INVALID
+	0x0000: {valueType: 4, unit: 0},              // RAW   => NONE
```

Comment:
@ramonhpr already fixed these warnings [here](https://github.com/CESARBR/knot-babeltower/pull/76/commits/dc11e3e95e6ca6137354401daf2896b853c1c3f0). As it's near to be merged, I think we can drop this commit.

Reply:
Done!